README.md

```diff
@@ -89,7 +89,19 @@ bind mounts and exposing ports for you.
     
 Note that `<PROJECT_ROOT>` must be absolute.
 
-You will then be able to view the application on http://localhost:5000/    
+You will then be able to view the application on http://localhost:5000/
+
+### Testing (in docker)
+The `Dockerfile` has a special stage for running tests.
+
+To build:
+    
+     docker build --target test --tag <image>:<tag> .
+To run:
+
+    docker run --mount type=bind,source=<PATH_ON_LOCAL>,target=/app <image>:<tag> .
+    
+NOTE: As of right now, the e2e selenium tests do not work.
```

Comment:
I think this is slightly outdated, given that you have the end-to-end tests passing in travis. We should fix the above command up (by passing the `.env` file to the `--env-file` flag) so that they can be run via the README instructions

Reply:
removed that NOTE. Also, I run this without --env-file on my local fine, assume it's picked up automatically. So I will not add it.